petab/core.py

```diff
@@ -1,15 +1,17 @@
 """PEtab core functions (or functions that don't fit anywhere else)"""
 
 import logging
+import os
 from typing import Iterable, Optional, Callable, Union, Any
 from warnings import warn
 
+import libcombine
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'libcombine'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4966219)